src/lib.rs

```diff
@@ -79,7 +79,7 @@
 //! [feature]: https://doc.rust-lang.org/cargo/reference/manifest.html#the-features-section
 
 #[cfg(feature = "serde")]
-extern crate serde_crate;
+extern crate serde;
```

Comment:
This code can be removed. This is no longer needed in Rust with `edition = "2018"`, so the edition might have to be set. But can also happen in other PR.